### ISSUE TITLE
Gnome Terminal titlebar is not hidden

### ISSUE THREAD

**Issue by Ludonope** [velitasali/GTKTitleBar#8]
I've installed this extension an it works perfectly except for gnome-terminal.

Since Gnome 3.32 I tried a few different extensions, and I have this same problem with similar extensions (such as Unite).

I don't have the problem with any other program, and I tried to disable every other extension to make sure it was not a conflict.

I run on ArchLinux 5.0.7.

**Comment by velitasali** [comment]
But it is a GTK application and makes use of the title bar space especially after the version 3.32 (of GNOME Terminal). 

This might help also #6

**Comment by lightbulbjim** [comment]
Can confirm that this is a problem. `gsettings set org.gnome.Terminal.Legacy.Settings headerbar false` changes the header bar back to a title bar, but GTKTitleBar then fails to hide it.

**Comment by unlimitedsola** [comment]
worth to mention that the issue @lightbulbjim described only reproducible on Wayland.

**Comment by lightbulbjim** [comment]
Good point, I should have mentioned that I'm using Wayland. Haven't tested on Xorg.

**Comment by unlimitedsola** [comment]
@lightbulbjim btw, as they (gnome) removed the support for `hide_titlebar_when_maximized`  (ref: https://gitlab.gnome.org/GNOME/mutter/merge_requests/221), I don't think it's possible to remove Client-Side-Decoration in Wayland for now.

Edit: an alternative way to hide the title bar in Wayland is by utilizing CSS: https://joshtronic.com/2017/07/26/hide-title-bars-in-gnome-shell/

**Comment by lightbulbjim** [comment]
Hmm, oh well. For now I'll continue to use a titlebar for Gnome Terminal and hide it when maximised (using Unite at the moment, which works). Then eventually when I'm forced to use a headerbar I'll complain for a bit and then get over it ;).

**Comment by zachliu** [comment]
I'm using mint, things are bit special here :joy: 
```
 MMMMMMMMMMMMMMMMMMMMMMMMMmds+.        OS: Mint 19.2 tina
 MMm----::-://////////////oymNMd+`     Kernel: x86_64 Linux 4.15.0-58-generic
 MMd      /++                -sNMd:    Uptime: 3d 16m
 MMNso/`  dMM    `.::-. .-::.` .hMN:   Packages: 2594
 ddddMMh  dMM   :hNMNMNhNMNMNh: `NMm   Shell: zsh 5.4.2
     NMm  dMM  .NMN/-+MMM+-/NMN` dMM   Resolution: 3840x1080
     NMm  dMM  -MMm  `MMM   dMM. dMM   DE: Cinnamon 4.2.3
     NMm  dMM  -MMm  `MMM   dMM. dMM   WM: Muffin
     NMm  dMM  .mmd  `mmm   yMM. dMM   WM Theme: Mint-Y-Dark (Windows 10 Dark)
     NMm  dMM`  ..`   ...   ydm. dMM   GTK Theme: Adwaita-dark [GTK2/3]
     hMM- +MMd/-------...-:sdds  dMM   Icon Theme: Mint-Y
     -NMm- :hNMNNNmdddddddddy/`  dMM   Font: Ubuntu 10
      -dMNs-``-::::-------.``    dMM   CPU: Intel Core i5-8250U @ 8x 3.4GHz [25.0°C]
       `/dMNmy+/:-------------:/yMMM   GPU: Mesa DRI Intel(R) UHD Graphics 620 (Kabylake GT2)
          ./ydNMMMMMMMMMMMMMMMMMMMMM   RAM: 8999MiB / 15751MiB
             \.MMMMMMMMMMMMMMMMMMM
```
The `gsettings set org.gnome.Terminal.Legacy.Settings headerbar "@mb false"` command doesn't work. Nor is the [CSS solution](https://joshtronic.com/2017/07/26/hide-title-bars-in-gnome-shell/). My work around (thanks to this [link](https://gitlab.gnome.org/GNOME/mutter/merge_requests/221) that @unlimitedsola provided) is to put this in my `.zshrc`:
```
if [ "$TERM" = "xterm-256color" ]; then
  xprop \
    -id $(xdotool getactivewindow) \
    -f _MOTIF_WM_HINTS 32c \
    -set _MOTIF_WM_HINTS "0x2, 0x0, 0x0, 0x0, 0x0"
fi
```
which will hide that hideous title bar every time I start a gnome-terminal

**Comment by paulodiovani** [comment]
> The gsettings set org.gnome.Terminal.Legacy.Settings headerbar "@mb false" command doesn't work.

@zachliu 
Doesn't Mint have `gconftool` instead? I think it works (fairly) in the same way.

**Comment by zachliu** [comment]
> > The gsettings set org.gnome.Terminal.Legacy.Settings headerbar "@mb false" command doesn't work.
> 
> @zachliu
> Doesn't Mint have `gconftool` instead? I think it works (fairly) in the same way.

@paulodiovani
Can you provide some more details here? How do I use `gconftool`?
